gallery/src/PageHeader/tartan-logotype.svg

```diff
@@ -0,0 +1 @@
,0,0,1-6.47-5.57H170.9V20c0-6.37-3.91-11-10-11S150.61,13.75,150.61,19.61Zm10.3-7A5.4,5.4,0,0,1,166.63,18H155.19A5.57,5.57,0,0,1,160.91,12.65Zm-91.34-.37,1.82,3.06a8.42,8.42,0,0,1,6-2.56c2.84,0,4.79,1.43,4.79,3.77v3A8.49,8.49,0,0,0,75.62,17c-3.34,0-7.11,1.93-7.11,6.58,0,4.49,3.77,6.71,7.11,6.71a8.69,8.69,0,0,0,6.61-2.68v2.18h4.44V16.34c0-5.44-4-7.33-8.51-7.33A12.24,12.24,0,0,0,69.57,12.28ZM73.09,23.6c0-2.23,1.88-3.66,4.4-3.66A6.08,6.08,0,0,1,82.4,22v3.08a6.08,6.08,0,0,1-4.91,2.1C75,27.22,73.09,25.79,73.09,23.6Zm-52.64-4c0,5.83,4,10.65,10.5,10.65s10.56-4.82,10.56-10.65S37.52,9,31,9,20.45,13.83,20.45,19.61ZM25,19.54c0-3.58,2.11-6.74,5.86-6.74s5.89,3.16,5.89,6.74-2.1,6.77-5.89,6.77S25,23.16,25,19.54ZM.63,15.17c0,4.53,4.19,5.45,7.7,6.2,2.49.55,4.64,1.18,4.64,2.94,0,1.51-1.35,2.47-4,2.47A10.57,10.57,0,0,1,2,23.93L0,27.12a12.48,12.48,0,0,0,8.75,3.14c5.49,0,8.49-2.73,8.49-6.29,0-4.78-4.35-5.78-7.9-6.54-2.41-.54-4.44-1.09-4.44-2.64s1.48-2.26,3.81-2.26a8.29,8.29,0,0,1,6,2.43l1.86-3.1A11.71,11.71,0,0,0,8.66,9C3.51,9,.63,11.86.63,15.17Zm131.09-2.75V9.66h-4.46V37.54h4.46V27.05a8.1,8.1,0,0,0,6.59,3.3c5.31,0,9.09-4,9.09-10.61s-3.78-10.58-9.09-10.58A8.23,8.23,0,0,0,131.72,12.42Zm.12,11.16v-8a6.62,6.62,0,0,1,5.24-2.74c3.51,0,5.75,2.82,5.75,6.74s-2.24,6.77-5.75,6.77A6.75,6.75,0,0,1,131.84,23.58ZM50.35,12.41V9.66H45.93V29.8h4.42v-14A6.72,6.72,0,0,1,55.58,13c2.48,0,4.08,1,4.08,4.17V29.8h4.43V15.62c0-4.12-2.19-6.46-6.57-6.46A9.69,9.69,0,0,0,50.35,12.41ZM119.6,9.62l-5.94,15.13L107.72,9.62H103l8.34,20.44-1,2.3a2.77,2.77,0,0,1-2.91,1.75,4.61,4.61,0,0,1-1.68-.33l-.68,3.93a13.31,13.31,0,0,0,2.49.29c3.16,0,5.6-1.13,7-4.56l9.82-23.82Z"/><path class="cls-2" d="M186.92,31.08a6,6,0,0,0,4.27-1.34l-1.08-3.41a2.9,2.9,0,0,1-2,.73c-1.29,0-2-1-2-2.46V13.67h4.23v-4H186.1V4h-4.53V9.7h-3.45v4h3.45v12C181.57,29.18,183.43,31.08,186.92,31.08Zm13.5,0a8.77,8.77,0,0,0,6.72-2.76v2.24h4.53V16.74c0-5.61-4-7.56-8.67-7.56a12.36,12.36,0,0,0-8.75,3.37l1.85,3.15a8.52,8.52,0,0,1,6.17-2.63c2.89,0,4.87,1.47,4.87,3.89v3.11a8.6,8.6,0,0,0-6.72-2.68c-3.41,0-7.25,2-7.25,6.78C193.17,28.79,197,31.08,200.42,31.08ZM202.18,28c-2.54,0-4.44-1.47-4.44-3.71s1.9-3.76,4.44-3.76a6.07,6.07,0,0,1,5,2.16v3.15A6.07,6.07,0,0,1,202.18,28Zm19.88,2.59V16.35a7.19,7.19,0,0,1,5.35-2.76,6.42,6.42,0,0,1,1.42.13V9.23a8.89,8.89,0,0,0-6.77,3.54V9.7h-4.53V30.56Zm16.69.52A6,6,0,0,0,243,29.74l-1.08-3.41a2.94,2.94,0,0,1-2,.73c-1.29,0-2-1-2-2.46V13.67h4.22v-4h-4.22V4H233.4V9.7H230v4h3.45v12C233.4,29.18,235.26,31.08,238.75,31.08Zm13.49,0A8.79,8.79,0,0,0,259,28.32v2.24h4.53V16.74c0-5.61-4.05-7.56-8.67-7.56a12.36,12.36,0,0,0-8.75,3.37l1.85,3.15a8.52,8.52,0,0,1,6.17-2.63C257,13.07,259,14.54,259,17v3.11a8.6,8.6,0,0,0-6.73-2.68c-3.4,0-7.24,2-7.24,6.78C245,28.79,248.84,31.08,252.24,31.08ZM254,28c-2.54,0-4.44-1.47-4.44-3.71s1.9-3.76,4.44-3.76a6.07,6.07,0,0,1,5,2.16v3.15A6.07,6.07,0,0,1,254,28Zm19.88,2.59V16a6.84,6.84,0,0,1,5.35-2.81c2.54,0,4.18,1.08,4.18,4.32v13H288V15.88c0-4.28-2.25-6.7-6.73-6.7a9.85,9.85,0,0,0-7.33,3.37V9.7h-4.53V30.56Z"/></svg>
```

Comment:
I'm not sure how much it really matters since we have the `alt` tag, but it might be nice if this svg had a [`<title>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) containing its text.

Reply:
> I'm not sure how much it really matters since we have the alt tag, but it might be nice if this svg had a <title> containing its text.

Oh or maybe not as it sounds like that'll generate a tooltip